packages/amplify-provider-awscloudformation/package.json

```diff
@@ -79,7 +79,8 @@
     "@types/lodash.throttle": "^4.1.6",
     "@types/node": "^12.12.6",
     "@types/uuid": "^8.0.0",
-    "jest": "^29.5.0"
+    "jest": "^29.5.0",
+    "typescript": "^5.1.3"
```

Comment:
We already depend on `4.9.5` in the root package.json. Is this needed? If so, can we use `4.9.5` instead?

Reply:
we need to add it to each package if you want to execute yarn commands that call tsc from within the package directory, updated the version to match the rest of the project